src/reporter/query_1T1E1A.py

```diff
@@ -70,8 +70,7 @@ def query_1T1E1A(attr_name,   # In Path
     if err == "AggrMethod cannot be applied":
         r = {
             "error": "AggrMethod cannot be applied",
-            "description": "AggrMethod cannot be applied on type TEXT and BOOLEAN."
-        }
+            "description": "AggrMethod cannot be applied on type TEXT and BOOLEAN."}
```

Comment:
I noticed the brackets throughout these changes have been moved one line back. Was this done by hand or automatically? Because I think it was "correct" before.

Minor note though

Reply:
yep, automated linter